sycl/test-e2e/ESIMD/slm_block_load_store.cpp

```diff
@@ -107,8 +102,8 @@ int main() {
   // which has been fixed only in "1.3.26816", and in win/opencl version going
   // _after_ 101.4575.
   if (!esimd_test::isGPUDriverGE(Q, esimd_test::GPUDriverOS::LinuxAndWindows,
-                                 "26816", "101.4576")) {
-    std::cout << "Skipped. The test requires GPU driver 1.3.26816 or newer.\n";
+                                 "26690", "101.4827")) {
```

Comment:
That is odd that the test passes with older driver(26690) than the previously requested one(26816), but if that really passes, then fine. 

nit: The comment above is not consistent with the new changes/driver-checks.